CHANGELOG.md

```diff
@@ -371,3 +371,7 @@ Released with 1.0.0-beta.37 code base.
 - Bump `elliptic` from `6.5.3` to `6.5.4` for `web3-bzz` (#3940)
 - Bump `elliptic` from `6.5.3` to `6.5.4` for `web3-core-requestmanager` (#3945)
 - Rewrite `web3-eth-iban` in ES6 (#3955)
+
+### Removed
+
+- removed bzz and shh api
```

Comment:
So we'd probably want to keep all of the listed changes, besides this one, listed under `1.3.5` for consistency with `1.x`

The `3.x` section should only contain:
```
### Removed

- removed bzz and shh api
```

Reply:
🤙